src/components/oak.py

```diff
@@ -38,8 +38,7 @@
     handle_synced_color_and_depth,
     make_metadata_from_seconds_float,
 )
-from src.components.helpers.config import (
-    Validator,
+from src.config import (
```

Comment:
These OAK component changes happened because I refactored `Validator` as a class in favor of adding a class-scoped `validate` function for every native config type